src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -151,30 +140,43 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
     [amountAvailableCryptoPrecision, assetMarketData.price],
   )
 
-  const getOutboundFeeCryptoBaseUnit = useCallback(async (): Promise<Result<
-    string,
-    string
-  > | null> => {
-    if (!accountId) return null
+  const getOutboundFeeCryptoBaseUnit = useCallback(
+    async (
+      _quote?: Result<ThorchainSaversWithdrawQuoteResponseSuccess, string>,
```

Comment:
Not sure if there's a better naming for this, but basically either pass an *actual* quote in, meaning we don't need to do the 10,000 bps dance if the quote is: ok, or if no quote is passed or the quote is an :err, try again with 10,000 bps